addons/im_livechat/controllers/channel.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.http import request, route
+from odoo.addons.mail.controllers.discuss.channel import ChannelController
+
+
+class ChannelControllerLiveChat(ChannelController):
+    @route()
+    def discuss_channel_info(self, channel_id):
+        info = super().discuss_channel_info(channel_id)
+        if info or not request.env.user.has_group("im_livechat.im_livechat_group_manager"):
+            return info
+        channel = request.env["discuss.channel"]._get_from_context_or_raise((channel_id))
+        channel_info = channel._channel_info()
+        if len(channel_info) == 1:
+            channel_info[0]["readonly"] = True
+        return channel_info
```

Comment:
Too much/strange override, we should better design base code to work around this case.

Reply:
This is a minor update of the base method but it makes the override significantly prettier:
https://github.com/odoo/odoo/pull/137623/files#diff-ec9b6d3d5a022b3d96672121c5850b4a4b876c2f8f4dd0ac6f5b9266d310eaccL802